package.json

```diff
@@ -85,6 +87,7 @@
     "coveralls": "^3.1.1",
     "cypress": "^12.10.0",
     "cypress-image-snapshot": "^4.0.1",
+    "d3-sankey": "^0.12.3",
```

Comment:
It Might be better to install these dependencies inside `packages/mermaid` instead of root one.

Reply:
How to do it? I am not a JS developer (ha-ha), I am not well aware it. Will installing package in packages/mermaid help?

Reply:
> I am not a JS developer (ha-ha)

Neither do I, I just started learning it few months ago.

---

You could change directory to that directory and install it, pnpm would install it their:

```
cd packages/mermaid
pnpm install <package-name>
```

Or using `--filter` option:

```bash
pnpm install --filter=mermaid <package-name>
```

Reply:
BTW, as two "non-JS" devs, both of you are writing top-notch code 🚀

Reply:
Done